build.gradle

```diff
@@ -17,7 +17,7 @@ allprojects {
   apply plugin: 'android-sdk-manager'
 
   group = 'io.realm'
-  version = new File("${rootDir}/version.txt").text
+  version = new File("${rootDir}/version.txt").text.replaceAll("\\r|\\n","")
```

Comment:
I tried initially and it failed. At first I thought it was perhaps trim() wasn't available: http://www.groovy-lang.org/gdk.html, but after remembering to use it in all 3 projects, it does actually work, so much better than custom regexp.